civitai-downloader-v2/src/api/services/database_service.py

```
@@ -0,0 +1,270 @@
+                    sql += " AND m.type = ?"
+                    params.append(model_type)
+                    
+                sql += " ORDER BY d.downloaded_at DESC"
+                
+                cursor.execute(sql, params)
+                
+                # Convert to dict list
+                columns = [description[0] for description in cursor.description]
+                results = []
+                for row in cursor.fetchall():
+                    results.append(dict(zip(columns, row)))
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)